### PR TITLE
Removed references to the old front-end

### PR DESCRIPTION
**Pull request by GabriFila**
# Description

This PR removes all the code and references of the old front-end, specifically:
- remove the `/webservice` directory
- remove the `frontend-lint` job
- change the automatic labeling for the _sig/ui_ to the `/dashboard` directory

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -49,32 +49,6 @@ jobs:
           LINTLY_COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
 
 
-  frontend-lint:
```
Comment (giorio94) [inline .github/workflows/lint.yml:52]: In your opinion, would it be possible to keep this linting for the dashboard folder (obviously without checking out the submodule)?

**Review by giorio94 — APPROVED** [review]
Thank @GabriFila!

**Review by palexster — APPROVED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (GabriFila) [reply .github/workflows/lint.yml:52]: Actually using the submodule is a little bit tricky and I don't know if it the best thing to do, I would wait on that for the moment

**Review by frisso — APPROVED** [review]

**Pull request merged**